README.md

```diff
@@ -7,9 +7,11 @@ A demo of Python's project with automated tests.
 How to install?
 ===============
 1. Create separate virtual environment with Python 3.5.x.
-2. Run `pip install -r requirements.txt` to install required Python's dependencies.
+2. Run `pip install -r requirements.txt` or `pip install -r requirements-dev.txt`  to install required Python's dependencies.
```

Comment:
`How to install?` section is about automated tests. Why do you add development details here?